apis/r/R/getData.R

```diff
@@ -0,0 +1,38 @@
+##' Access a DataFrame or Single Column From a Given URI
+##'
+##' These functions access a given SOMA URI and extract, respectively, the
+##' complete data.frame or named column from the data set.
+##'
+##' @param uri Character value with URI path to a SOMA data set
+##' @param column Character value with the name of the column to retrieve
+##' @return The selected data frame (or column) from the given data set
+##' @examples
+##' \dontrun{
+##' uri <- "test/soco/pbmc3k_processed/obs"
+##' column <-  "n_counts"
+##' summary(getColumn(uri, column)
+##' summary(getTable(uri))
+##' }
+##' @export
```

Comment:
```suggestion
##' @noRd
```

Maybe keep these as internal functions for now since they're likely to change?

Reply:
Yes -- all this with likely change as soon as you and I have a change to sort out what we want to call and how.

I think I will start standardizing under a function `read()` maybe with a helper function `read_all()`.